modules/monitoring-enabling-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * monitoring/user-workload-monitoring.adoc
+
+[id="enabling-user-workload-monitoring_{context}"]
+= Enabling user workload monitoring
+
+You can enable user workload monitoring using the cluster monitoring ConfigMap.
```

Comment:
```suggestion
You can enable user workload monitoring by setting the ` techPreviewUserWorkload/enabled` flag in the cluster monitoring ConfigMap.
```

Reply:
We should specify the full name of the ConfigMap as well.